numba/core/interpreter.py

```diff
@@ -79,7 +79,38 @@ def get_assignment_source(self, destname):
         return None
 
 
-def _call_function_ex_replace_kws_small(keyword_expr, new_body, buildmap_idx):
+def _remove_assignment_definition(old_body, idx, func_ir):
+    """
+    Deletes the definition defined for old_body at index idx
+    from func_ir. We assume this stmt will be deleted from
+    new_body.
+    """
+    lhs = old_body[idx].target.name
+    rhs = old_body[idx].value
+    if rhs in func_ir._definitions[lhs]:
+        func_ir._definitions[lhs].remove(rhs)
+    elif len(func_ir._definitions[lhs]) == 1:
+        # There appears to be an issue where the RHS of the
+        # definitions doesn't match after the list -> tuple
+        # conversion.
+        # For example the statement is:
+        #
+        #   172list_to_tuple.116 = $170list_append.115
+        #
+        # but the definition is:
+        #
+        #   list_to_tuple(info=('$4build_list.1',))
+        #
+        func_ir._definitions[lhs].clear()
+    else:
+        raise UnsupportedError(
+            "Inconsistency found the in definitions trying to remove varkwargs"
```

Comment:
Is the error message still accurate? This function seem to be applied generally not just on varkwargs.

Reply:
How is this message? Also thank you for the ping, I missed the original review.